### ISSUE TITLE
Noisy activate/deactivate scripts on Windows

### ISSUE THREAD

**Issue by SuaveFool** [conda-forge/libxml2-feedstock#86]
### Comment:

While this isn't an actual error/problem, the activate/deactivate batch scripts for Windows don't prefix the commands with @, this means that activating/deactivating environments with libxml2 display the set commands on the command line. Can this be changed to prefix the SET with an @, like: 

activate.bat
```BATCH 
@set xml_catalog_files_libxslt=%XML_CATALOG_FILES%
@set XML_CATALOG_FILES=%CONDA_PREFIX%\etc\xml\catalog
```
deactivate.bat
```BATCH
@if defined xml_catalog_files_libxslt (
    set XML_CATALOG_FILES=%xml_catalog_files_libxslt%
) else (
    set XML_CATALOG_FILES=
)
@set xml_catalog_files_libxslt=
```

Other libraries that have quiet activate/deactivate scripts that I'm using are:
https://github.com/conda-forge/gdal-feedstock/blob/main/recipe/scripts/activate.bat
https://github.com/conda-forge/proj.4-feedstock/blob/main/recipe/scripts/activate.bat

**Comment by hattne** [comment]
Should be fixed as of #91.

**Comment by jakirkham** [comment]
It looks like this was incorporated in PR: https://github.com/conda-forge/libxml2-feedstock/pull/88

Going to close this as resolved
